app/Http/Controllers/Agent/helpdesk/MailController.php

```diff
@@ -356,6 +356,14 @@ public function get_data(Request $request)
     {
         $id = $request->input('image_id');
         $attachment = \App\Model\helpdesk\Ticket\Ticket_attachments::where('id', '=', $id)->first();
+        $current_user = \Auth::user();
```

Comment:
When the Ticket Fetch command is running, at that time, Auth::user() will cause an issue.